### PR TITLE
remontoire: init at 20220619

### PR DESCRIPTION
**Pull request by aacebedo**
## Description of changes

add remontoire from regolith linux

## Things done

- Built on platform(s)
  - [X] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [23.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) (or backporting [23.05 Release notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2305.section.md))
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/misc/remontoire/default.nix [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+, vala
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
+  pname = "remontoire";
+  version = "20220619";
+
+  src = fetchFromGitHub {
+    owner = "regolith-linux";
+    repo = pname;
```
Comment (AndersonTorres) [inline pkgs/applications/misc/remontoire/default.nix:24]:
```suggestion
    repo = "remontoire";
```

File: pkgs/applications/misc/remontoire/default.nix [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+, meson
+, ninja
+, pkg-config
+, python3
+, vala
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
+  pname = "remontoire";
+  version = "20220619";
```
Comment (AndersonTorres) [inline pkgs/applications/misc/remontoire/default.nix:20]:
```suggestion
  version = "unstable-2022-06-19";
```

File: pkgs/applications/misc/remontoire/default.nix [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+
+  src = fetchFromGitHub {
+    owner = "regolith-linux";
+    repo = pname;
+    rev = "68d562c78d6e0094ca744bd7161c308f583e93e";
+    sha256 = "Cb6tzTGZdQA9oA04DO/xLBw5F+FRj5BM2Aa62YWGmZA=";
```
Comment (AndersonTorres) [inline pkgs/applications/misc/remontoire/default.nix:26]:
```suggestion
    hash = "sha256-Cb6tzTGZdQA9oA04DO/xLBw5F+FRj5BM2Aa62YWGmZA=";
```

File: pkgs/applications/misc/remontoire/default.nix [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+    libgee
+  ];
+
+  postPatch = ''
+    chmod +x build-aux/meson/postinstall.py
+    patchShebangs build-aux/meson/postinstall.py
+  '';
+
+  meta = with lib; {
+    description = "Remontoire is a small GTK app for presenting keybinding hints";
```
Comment (AndersonTorres) [inline pkgs/applications/misc/remontoire/default.nix:52]:
```suggestion
    description = "A small GTK app for presenting keybinding hints";
```

File: pkgs/applications/misc/remontoire/default.nix [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+    homepage = "https://github.com/regolith-linux/remontoire";
+    license = licenses.gpl3;
```
Comment (AndersonTorres) [inline pkgs/applications/misc/remontoire/default.nix:54]:
```suggestion
    license = licenses.gpl3Plus;
```

**Review by AndersonTorres — CHANGES_REQUESTED** [review]

File: pkgs/applications/misc/remontoire/default.nix [hunk 6]
```diff
@@ -0,0 +1,58 @@
...
+, libgee
+, librsvg
+, meson
+, ninja
+, pkg-config
+, python3
+, vala
+}:
+
+stdenv.mkDerivation (finalAttrs: rec {
```
[the remark this answers is not in the document]
Comment (aacebedo) [reply pkgs/applications/misc/remontoire/default.nix]: done

File: pkgs/applications/misc/remontoire/default.nix (hunk 1 above)
Comment (aacebedo) [reply pkgs/applications/misc/remontoire/default.nix]: done

File: pkgs/applications/misc/remontoire/default.nix (hunk 2 above)
Comment (aacebedo) [reply pkgs/applications/misc/remontoire/default.nix]: done

File: pkgs/applications/misc/remontoire/default.nix (hunk 3 above)
Comment (aacebedo) [reply pkgs/applications/misc/remontoire/default.nix]: done

File: pkgs/applications/misc/remontoire/default.nix (hunk 5 above)
Comment (aacebedo) [reply pkgs/applications/misc/remontoire/default.nix]: done

File: pkgs/applications/misc/remontoire/default.nix (hunk 4 above)
Comment (aacebedo) [reply pkgs/applications/misc/remontoire/default.nix]: done

**Comment by aacebedo** [comment]
@AndersonTorres can you give another check

**Review by AndersonTorres — APPROVED** [review]

**Comment by aacebedo** [comment]
Is it possible to merge it?

**Pull request merged**